Zend/zend_attributes.c

```diff
@@ -146,6 +147,48 @@ ZEND_METHOD(Override, __construct)
 	ZEND_PARSE_PARAMETERS_NONE();
 }
 
+ZEND_METHOD(Deprecated, __construct)
+{
+	zend_string *message = NULL;
+	zend_string *since = NULL;
+	zend_string *property_name;
+	zval value;
+
+	ZEND_PARSE_PARAMETERS_START(0, 2)
+		Z_PARAM_OPTIONAL
+		Z_PARAM_STR_OR_NULL(message)
+		Z_PARAM_STR_OR_NULL(since)
+	ZEND_PARSE_PARAMETERS_END();
+
+	property_name = ZSTR_KNOWN(ZEND_STR_MESSAGE);
+	if (message) {
+		ZVAL_STR(&value, message);
+	} else {
+		ZVAL_NULL(&value);
+	}
+	zend_update_property_ex(zend_ce_deprecated, Z_OBJ_P(ZEND_THIS), property_name, &value);
+	zend_string_release(property_name);
```

Comment:
Indeed. I guess it was added for consistency, but I also prefer omitting no-ops.

Reply:
Yes, I added that for visual consistency.

-----------------

Currently waiting for the CI to look at the Closure change and then I'll push a commit to add known strings for both `Deprecated` and `since`, which will implicitly resolve this.